src/main/steps/help-with-your-fee/content.ts

```diff
@@ -25,13 +35,17 @@ const cy: typeof en = ({ isDivorce, required }) => ({
   line1: `Mae'r ${
     isDivorce ? 'cais am ysgariad' : 'cais'
   } hwn yn costio £550. Efallai y byddwch yn gallu cael help i dalu'r ffi:`,
+  line2: '',
```

Comment:
maybe best to do ...en() until these are translated. not a big deal tho.

Reply:
```suggestion
    ...en({ isDivorce, required, isJointApplication, partner }),
```

Reply:
but have this above the other properties

Reply:
thanks!